docker/scripts/delete-indexed-documents

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+SCRIPT_NAME=$(basename "$0")
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME --all
+   or: $SCRIPT_NAME COLLECTION... (for example: $SCRIPT_NAME artist release)
+   or: $SCRIPT_NAME --help
+
+For all or each of the given MusicBrainz Solr collections,
+delete the indexed documents.
+
+Pre-requisites:
+  Solr must be running (in SolrCloud mode).
+EOH
+)
+
+# Parse arguments
+
+if [ $# -eq 0 ]
+then
+	echo >&2 "$SCRIPT_NAME: missing argument"
+	echo >&2 "Try '$SCRIPT_NAME --help' for usage."
+	exit 64 # EX_USAGE
+elif [[ "$*" =~ ^-*h(elp)?$ ]]
+then
+	echo "$HELP"
+	exit 0 # EX_OK
+elif [[ $# -eq 1 && $1 =~ ^all$ ]]
+then
+	collections=()
+else
+	collections=("$*")
+fi
+
+# Check pre-requisites
+
+max_try=${MAX_TRY:-12}
+wait_seconds=${WAIT_SECONDS:-5}
+local_tries=1
+while [[ -f $SOLR_HOME/musicbrainz-solrcloud-first-run && $local_tries -lt $max_try ]]
+do
+	local_tries=$((local_tries + 1))
+	sleep "$wait_seconds"
+done
+if [[ -f $SOLR_HOME/musicbrainz-solrcloud-first-run && $local_tries -eq $max_try ]]
```

Comment:
Is it possible for the second condition to be false?
```suggestion
if [[ -f $SOLR_HOME/musicbrainz-solrcloud-first-run ]]
```

Reply:
No. I moved this condition as comment for clarity, through the amending commit https://github.com/metabrainz/mb-solr/pull/61/commits/fd512744979edc0734cd3797c8b0497be315a853.